native/libp2p_port/internal/subscriptions/subscriptions.go

```diff
@@ -28,6 +29,79 @@ type Subscriber struct {
 	port            *port.Port
 }
 
+type GossipTracer struct {
+	port *port.Port
+}
+
+func (g GossipTracer) AddPeer(p peer.ID, proto protocol.ID) {
+	// no-op
+}
+
+func (g GossipTracer) RemovePeer(p peer.ID) {
+	// no-op
+}
```

Comment:
These two are the most important. Could you add them on this PR?

Reply:
Done